src/execution/operator/projection/physical_tableinout_function.cpp

```diff
@@ -87,6 +96,8 @@ OperatorResultType PhysicalTableInOutFunction::Execute(ExecutionContext &context
 		state.input_chunk.SetCardinality(1);
 		state.row_index++;
 		state.new_row = false;
+		lock_guard<mutex> guard(gstate.ordinality_lock);
+		gstate.current_ordinality_idx = 1;
```

Comment:
Given that the other scan state is all located in the local state - I suspect the global state is the wrong place for the `current_ordinality_idx`. Do we have a test with a larger `UNNEST` or similar table function that triggers multi-threading?